CHANGELOG.md

```diff
@@ -12,6 +12,128 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 🛠 Maintenance
 ## 📚 Documentation -->
 
+# [0.0.5] - 2021-03-30
+
+> Important: X breaking changes below, indicated by **❗ BREAKING ❗**
+
+## 🚀 Features
+- **Adds introspection ability for subgraphs - [lrlna], [issue/349] [pull/377]**
+
+  A new command, `rover subgraph introspect` has been added. This command
+  runs a _federated introspection_ query against a server which has
+  implemented the requirements of the [federation
+  specification](https://www.apollographql.com/docs/federation/federation-spec/).
+  This command accepts endpoint [headers] for making the introspection
+  request (if required) and outputs SDL to stdout.
+
+  [lrlna]: https://github.com/lrlna
+  [pull/377]: https://github.com/apollographql/rover/pull/277
+  [issue/349]: https://github.com/apollographql/rover/issues/349
+
+- ** Fallback to monochromic output when `tput` is unavailable - [abernix], [issue/371] [pull/372]**
+
+  The `tput` command allows easier ANSI output using named values in rather than
+  control characters.
+
+  While we could just use control characters and maintain colored output in the
+  absence of `tput`, it's probably also reasonable to gracefully fall back to
+  monochromatic output.
+
+  [abernix]: https://github.com/abernix
+  [pull/372]: https://github.com/apollographql/rover/pull/372
+  [issue/371]: https://github.com/apollographql/rover/issues/371
+
+## ❗ BREAKING ❗
+- ** Renames `core build` to `supergraph compose` - [lrlna], [pull/391]**
+
+  To align with other Apollo teams on the usage `supergraph` and
+  `composition`, we are renaming `core build` to `supergraph compose`.
+
+  [lrlna]: https://github.com/lrlna
+  [pull/391]: https://github.com/apollographql/rover/pull/391
+
+- ** Updates harmonizer@0.2.2 - [abernix], [pull/396]**
+
+  Updates harmonizer to the latest version. This version now composes and
+  returns a core schema instead of CSDL. CSDL was an internal implementation
+  of composition and this new format is meant to bring stability to `rover
+  supergraph compose.
+
+  [abernix]: https://github.com/abernix
+  [pull/396]: https://github.com/apollographql/rover/pull/396
+
+## 🐛 Fixes
+- ** handle 400-599 HTTP responses - [lrlna], [issue/394] [issue/187] [pull/395]**
+
+  We are not currently handling any unsuccessful requests wehn handling a
+  response from an endpoint. This fix makes sure we check for errors to
+  return those errors before trying to parse the response body.
```

Comment:
```suggestion
  Previously, Rover did not provide errors for any HTTP requests that return a status code between 400 and 599. This fix makes sure Rover checks for those errors before moving on to parsing the response body.
```

Reply:
I _personally_ think this belongs squarely in the feature section! Better error messages feel like features to me. Not a hill I'll die on though :)